api/app/routers.py

```diff
@@ -159,3 +160,80 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):  # type: ignor
             if db != "default":
                 return db == "analytics"
         return None
+
+
+class TaskProcessorRouter:
+    """
+    Routing of database operations for task processor models
+
+    This router is used if TASK_PROCESSOR_DATABASE_* settings are defined.
+    """
```

Comment:
@Flagsmith/flagsmith-back-end It makes sense for this router to live here, because the behavior is demanded by the API specifically. However, because it looks very standard for now, it feels like it could be offered by flagsmith-common. What's the team opinion on this? I'm happy to go either path, though I believe testing and docs would be easier if contained in flagsmith[-core].

Reply:
Yup, moved to flagsmith-common (https://github.com/Flagsmith/flagsmith-common/pull/68/commits/ec6c2e5756967a83d5bfe47f72c3d1945e646e8e), as it was actually necessary. I will update this PR next time I'm working on this, likely on Monday, and resolve this thread afterwards.